lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -2368,6 +2368,108 @@ setReadOnly( Complex64Array.prototype, 'some', function some( predicate, thisArg
 	return false;
 });
 
+/**
+* Sorts an array in-place.
+*
+* @name sort
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Function} [compareFcn] - comparison function
```

Comment:
```suggestion
* @param {Function} compareFcn - comparison function
```

The function is mandatory.